.github/dependabot.yml

```diff
@@ -5,3 +5,7 @@ updates:
   schedule:
     interval: daily
   open-pull-requests-limit: 10
+  reviewers:
+    - Revolyssup
+    - DelusionalOptimist
+    - utkarsh-pro
```

Comment:
Did each of these individuals confirm their assignment?

Reply:
Yes

Reply:
Wonderful. Thank you for getting us organized. ✔️